packages/asini-changelog/.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
`node_modules` are ignored by default, and eslint should consult `.gitignore` files as well?